tests/test_recipient_validation.py

```diff
@@ -323,51 +323,6 @@ def test_validate_email_address_raises_for_invalid(email_address):
     assert str(e.value) == 'Not a valid email address'
 
 
-@pytest.mark.parametrize('column', [
-    'address_line_1',
-    'address_line_2',
-    'address_line_3',
-    'address_line_4',
-    'address_line_5',
-    'address_line_6',
-    'postcode',
-])
-def test_validate_address_doesnt_raise_for_missing_optional_columns(column):
-    assert validate_recipient('', 'letter') == ''
-
-
-@pytest.mark.parametrize('column', [
-    'address_line_1',
-    'address_line_2',
-    'address_line_3',
-    'address_line_4',
-    'address_line_5',
-    'address_line_6',
-    'postcode',
-])
-def test_validate_address_allows_any_non_empty_value_for_all_columns(column):
-    assert validate_recipient('any', 'letter') == 'any'
-
-
-@pytest.mark.parametrize('column', [
-    'address_line_1',
-    'address_line_2',
-    'address_line_3',
-    'address_line_4',
-    'address_line_5',
-    'address_line_6',
-    'postcode',
-])
-def test_non_ascii_address_line_is_fine_for_all_columns(column):
```

Comment:
Is there some sort of check that we still want to ensure non ascii content in any of the address lines are fine?